### DISCUSSION TITLE
Phone Verification for FBW Discord Server

### DISCUSSION THREAD

**Discussion by OddSobriquet** [flybywiresim/aircraft#2458] [General]
I have a question regarding the Discord server:
Why do I suddenly need to verify my phone number to send messages on the discord server?
I fail to understand why either discord, fly by wire or any of the bot's creator's should be in possession of my phone number.
It is not mentioned anywhere in the rules channel, nothing in the FAQ, nothing in welcome, there is no announcement in Server announcements either.

**Comment by hotshotp** [comment]
this is discord - its called 2 factor authentication and means that there will be less spammers. flybywire has no notice of your phone number

**Reply by OddSobriquet** [reply]
weirdly enough this is the only server of the 15+ I'm on that has that.

**Reply by hotshotp** [reply]
yes every big discord server or server that doesnt like spam does 2fa, its really not a new thing

**Reply by OddSobriquet** [reply]
well, it's new to me, especially since I have never encountered it on all my time being on discord no matter how big the servers were...
Since I'm not a fan of just handing out my personal contact details willy nilly I will be leaving the server. It's a bit of a shame but alright.

**Reply by pepperoni505** [reply]
Just to clarify some things, your phone number is only shown to you. No one at FBW can see your private information. We require a certain level of verification in order to filter out trolls. If you want to, check out Discord's privacy policy here: https://discord.com/privacy

**Reply by ghost** [reply]
Yes, as others have said, this is to just prevent spam/trolling/ban evasions. No one at FBW is able to see your number, as it is discord that handles this.

**Reply by OddSobriquet** [reply]
> have you never put in your phone number before, seriously youre a bit of an *** if you think you will leave the server, what a ******* spark

classy reply. While, admittedly, stern I tried my best to stay polite and neutral to anyone who was involved in this decision. I didn't make it about them or their personalities.
There are a few details I won't hand out without a valid and proper reason. My phone number is one of them and access to a collection of chat rooms is not on the list of valid and proper reasons. It simply isn't proportionate in my book. If it is in yours, that's fine, but not everyone applies your standards to their lives. You may not like it or be on the same page with others and their decisions but calling people an ass because of this crosses a bit of a line.

**Reply by ghost** [reply]
@propilot43 We do not tolerate abuse and insults on our GitHub or Discord server.

**Reply by Benjozork** [reply]
@OddSobriquet we sincerely apologize for the words used by that user. They are in no way a team member nor do they represent FlyByWire, and appropriate action has been taken by the moderation team.

**Reply by OddSobriquet** [reply]
> @OddSobriquet we sincerely apologize for the words used by that user. They are in no way a team member nor do they represent FlyByWire, and appropriate action has been taken by the moderation team.

Don't worry, I'm not holding you accountable for their actions. :)

**Comment by ghost** [comment]
name and shame that guy on discord, im done

**Comment by Benjozork** [comment]
Phone number authentication is a Discord feature and we do not have access to any of that information. We have enabled it for the time being after doxxing and hacks occured on our Discord server.
